src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -150,6 +151,25 @@ async function getAdditionalFacilityInfo(futureAppointments, useV2 = false) {
   return facilityData;
 }
 
+/**
+ * Function to retrieve facility information from the appointment
+ * record when using the v2 api.
+ *
+ * @param {*} appointments
+ */
+function getAdditionalFacilityInfoV2(appointments) {
+  // Facility information included with v2 appointment api call.
+  const facilities = appointments
+    ?.map(
+      appt =>
+        appt.vaos.apiData.location
+          ? { ...appt.vaos.apiData.location.attributes }
+          : null,
+    )
+    .filter(n => n);
+  return facilities ? transformFacilitiesV2(facilities) : [];
```

Comment:
It seems like this whole function could be `return appointments.map(appt => appt.vaos.facilityData).filter(n => n)`. I don't think you need the transform here, since it's in the appointment service.